packages/libtinfo.rb

```diff
@@ -18,8 +18,8 @@ class Libtinfo < Package
   def self.install
     # See https://forums.opensuse.org/showthread.php/446927-missing-library-libtinfo-so-5.
     # See also http://www.linuxforums.org/forum/installation/6251-libtinfo-so-5-a.html.
-    system "mkdir -p #{CREW_DEST_LIB_PREFIX}"
-    system "ln -sf #{CREW_LIB_PREFIX}/libncurses.so.6 #{CREW_DEST_LIB_PREFIX}/libtinfo.so.5"
-    system "ln -sf #{CREW_LIB_PREFIX}/libncurses.so.6 #{CREW_DEST_LIB_PREFIX}/libtinfo.so.6"
+    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}"
+    FileUtils.ln_sf "#{CREW_LIB_PREFIX}/libncurses.so.6 #{CREW_DEST_LIB_PREFIX}/libtinfo.so.5"
+    FileUtils.ln_sf "#{CREW_LIB_PREFIX}/libncurses.so.6 #{CREW_DEST_LIB_PREFIX}/libtinfo.so.6"
```

Comment:
I believe FileUtils requires a comma to separate the parameters, no?
Also, you could simplify:
```
FileUtils.mkdir_p CREW_DEST_LIB_PREFIX
```
No need to quote an isolated constant.

Reply:
Done